docs/scenarios.md

```diff
@@ -1,36 +1,34 @@
+*In each of the three scenarios lister above, {{< kafka-operator >}} does not use StatefulSet, relying, instead, on Pods, PVCs and ConfigMaps. With {{< kafka-operator >}}, StatefulSet is very convenient starting point, as it handles roughly 80% of scenarios but introduces huge limitations when running Kafka on Kubernetes in production.*
 
 ## Monitoring based control
 
-Use of monitoring is essential for any application, and all relevant information about Kafka should be published to a monitoring solution. When using Kubernetes, the de facto solution is Prometheus, which supports configuring alerts based on previously consumed metrics. We wanted to build a standards-based solution (Prometheus and Alert Manager) that could handle and react to alerts automatically, so human operators wouldn't have to. {{< kafka-operator >}} supports alert-based Kafka cluster management.
+Use of monitoring is essential for any application, and all relevant information about Kafka should be published to a monitoring solution. When using Kubernetes, the de facto solution is Prometheus, which supports configuring alerts based on previously consumed metrics. {{< kafka-operator >}} was built as a standards-based solution (Prometheus and Alert Manager) that could handle and react to alerts automatically, so human operators wouldn't have to. {{< kafka-operator >}} supports alert-based Kafka cluster management.
 
 ## LinkedIn's Cruise Control
 
-We have a lot of experience in operating both Kafka and Kubernetes at scale. However, we believe that LinkedIn knows how to operate Kafka even better than we do. They built a tool, called Cruise Control, to operate their Kafka infrastructure, and we wanted to build an operator which **handled the infrastructure but did not reinvent the wheel insofar as operating Kafka**. We didn't want to redevelop proven concepts, but wanted to create an operator which leveraged our deep Kubernetes expertise by handling all Kafka infrastructure related issues in the way we thought best. We believe managing Kafka is a separate issue, for which there already exist some unique tools and solutions that are standard across the industry, so we took LinkedIn's Cruise Control and integrated it with the operator.
+  LinkedIn knows how to operate Kafka in a better way. They built a tool, called Cruise Control, to operate their Kafka infrastructure. And {{< kafka-operator >}} is built to **handle the infrastructure, but not to reinvent the wheel in so far as operating Kafka**. {{< kafka-operator >}} is an operator which leveraged deep Kubernetes expertise by handling all Kafka infrastructure related issues in the best possible way. Managing Kafka can be a separate issue, for which there already exist some unique tools and solutions that are standard across the industry, so LinkedIn's Cruise Control is integrated with the {{< kafka-operator >}}.
```

Comment:
**Recommended**: The first sentence sounds like we were inferior to CC while that's not the case, we are complementary tools to each other, we **SHOULD** phrase this accordingly.

```suggestion
  LinkedIn knows how to operate Kafka at scale. They built a tool, called Cruise Control, to operate their Kafka infrastructure. And {{< kafka-operator >}} is built to **handle the infrastructure, but not to reinvent the wheel in so far as operating Kafka**. {{< kafka-operator >}} is an operator which leveraged deep Kubernetes expertise by handling all Kafka infrastructure related issues in the best possible way. Managing Kafka can be a separate issue, for which there already exist some unique tools and solutions that are standard across the industry, so LinkedIn's Cruise Control is integrated with the {{< kafka-operator >}}.
```